### ISSUE TITLE
Many ID swaps are seen in local MOT17 testing

### ISSUE THREAD

**Issue by jinmaodaomaye2021** [PeizeSun/TransTrack#40]
Hi author,

I am using the MOT17  pretrained model to reproduce the results in MOT17 test data. 

I visually checked results of "MOT17-01-FRCNN" and found that the performance is not as same as that in the MOT17 challenge. There are many ID swaps when persons walk cross each other. However, the ID swaps are not seen in the MOT17 challenge results. 

The command I am using is 

`python3.7 main_track.py --output_dir . --dataset_file mot --coco_path mot --batch_size 1 --resume model/619mot17_mot17.pth --track_eval_split test --eval --with_box_refine  --num_queries 500`

Could you please advise on the root cause ? Thanks.

**Comment by PeizeSun** [comment]
Hi~
This is because MOT17 challenge results is not from 619mot17_mot17.pth, it is from the model trained on mixture of crowdhuman and mot17. Please see test set part in README.md.
